release/preview/debian11/meta.json

```
@@ -0,0 +1,16 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion" : false,
+    "PackageFormat": "powershell-${PS_VERSION}-linux-x64.tar.gz",
+    "osVersion": "Debian 11",
```

We should say testing until it's released
```suggestion
    "osVersion": "Debian 11 (Testing)",
```